package.json

```diff
@@ -12,32 +12,29 @@
     "gitmoji-config": "gitmoji -g",
     "commit": "gitmoji -c",
     "type-check": "tsc -p ./tsconfig.json",
-    "test": "NODE_PATH=./ ts-mocha -p ./tsconfig.test.json ./testSetup.tsx src/testSetup.tsx src/**/*.test.tsx",
-    "test:watch": "NODE_PATH=./ ts-mocha --watch -p ./tsconfig.test.json ./testSetup.tsx src/testSetup.tsx src/**/*.test.tsx",
     "build::assets": "cp ./package.json ./dist/package.json && cp -r ./src/assets ./dist/assets && cp ./README.md ./dist/README.md",
     "build::ts": "tsc -p ./tsconfig.json",
     "build::babel": "babel ./dist --out-dir ./dist",
     "build::cleanup": "rm ./dist/**/*.jsx ./dist/*.jsx",
-    "build": "rm -rf ./dist && npm run build::ts && npm run build::babel && npm run build::assets && npm run build::cleanup",
+    "build": "rm -rf ./dist && npm run build::ts && npm run build::assets && npm run build::babel && npm run build::cleanup",
     "publish-uikit": "cd dist && npm publish --access public",
-    "prewatch": "rm -rf dist && mkdir dist && cd dist && npm link react react-dom",
-    "watch": "rm -rf ./dist && tsc-watch -p ./tsconfig.json --onSuccess \"npm run build::babel\"",
-    "postwatch": "rm -r dist"
+    "watch": "tsc-watch -p ./tsconfig.json --onSuccess \"npm run build::babel\"",
+    "postwatch": "rm -r dist",
+    "test": "jest src/ThemeProvider/*",
+    "test:watch": "npm run test -- --watch"
   },
   "peerDependencies": {
-    "react": "16.x.x || 17.x.x",
-    "react-dom": "16.x.x || 17.x.x"
```

Comment:
I may be wrong, but `react` and `react-dom` may still need to be peer deps, so that React doesn't throw mismatching versions for hooks. 
That's happened to me while using npm/yarn `link`.

Reply:
I think you're right. This has proven to be a big issue so I'm tackling this in a seperate PR.